doc/man3/SSL_CTX_set1_curves.pod

```diff
@@ -96,6 +100,9 @@ is -1.
 When called on a client B<ssl>, SSL_get_shared_group() has no meaning and
 returns -1.
 
+SSL_get_negotiated_group() returns the NID of the negotiated group on a
```

Comment:
Probably this needs adding to the HISTORY section below.

Reply:
Done